godtools/App/Share/Domain/UseCases/GetDeviceLanguageCodeUseCase.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  GetDeviceLanguageCodeUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/25/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+class GetDeviceLanguageCodeUseCase {
```

Comment:
Nice we are definitely needing this too.  I'm even thinking of phasing out DeviceLanguageType and moving that logic directly in here, but for now this is a start.